src/olympia/signing/tests/test_views.py

```diff
@@ -94,11 +99,80 @@ def test_addon_does_not_exist(self, sign_version):
         assert addon.status == amo.STATUS_LITE
         sign_version.assert_called_with(addon.latest_version, is_beta=False)
 
+    @mock.patch('olympia.devhub.views.auto_sign_version')
```

Comment:
since this decorator is used on so many methods, it might make more sense in setUp like:

    patcher = mock.patch(...)
    self.auto_sign_version = patcher.start()